subprojects/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginCheckInIntegrationTest.groovy

```diff
@@ -121,8 +124,36 @@ class GradleEnterprisePluginCheckInIntegrationTest extends AbstractIntegrationSp
         getMinimumPluginVersionForConfigurationCaching() | true
     }
 
+    @Requires(IntegTestPreconditions.NotConfigCached)
+    def "shows warning message when Gradle Enterprise plugin version is used with isolated projects enabled"() {
+        given:
+        plugin.runtimeVersion = plugin.artifactVersion = pluginVersion
```

Comment:
💅 I see that this is how it is in the existing CC test, but I would argue we should avoid this coding style and split assignments into separate statements